bee-network/bee-autopeering/src/time.rs

```diff
@@ -0,0 +1,35 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use std::time::{SystemTime, UNIX_EPOCH};
```

Comment:
We do have a time module in bee_common now, these are probably good candidates to add there

Reply:
I will check that.

Reply:
I replaced some logic with what bee-common already provides. Moving some logic from this crate to bee-common should be done in a separate PR.